locallibrary/catalog/migrations/0001_initial.py

```diff
@@ -0,0 +1,64 @@
+# Generated by Django 3.2.5 on 2021-07-30 01:55
```

Comment:
Chú ý các pull sau update model thì sẽ makemigrations để tạo ra file migrations version mới, chứ ko xóa hết toàn bộ migrations trước đó đâu e nha, như này sẽ không thể teamwork trong dự án được